backend/backend/AimTrainerGame/Models/Highscore.cs

```diff
@@ -0,0 +1,12 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace backend.AimTrainerGame.Models;
+
+public class Highscore
+{
+    [Key]
+    public Guid Id { get; set; }
+    public string Username { get; set; }
```

Comment:
Cia meta warninga. Reikia `required` pries `string`. Siaip not sure, bet, man atrodo, nepakenktu ir kitus `required` pazymet.

Reply:
Yep, sita sutvarkysiu